Source/spells.h

```diff
@@ -9,9 +9,21 @@
 
 namespace devilution {
 
+enum class SpellCheckResult : uint8_t {
+	Success = 0,

```

Comment:
To match other enum styles
```suggestion
	Success,
```